beta/nncf/tensorflow/sparsity/rb/algorithm.py

```diff
@@ -149,25 +151,26 @@ def raw_statistics(self):
         sparsity_rate_for_model = (total_sparsified_weights_number / model_weights_number).numpy()
         mean_sparse_prob = (sparse_prob_sum / tf.cast(total_weights_number, tf.float32)).numpy()
 
-        raw_sparsity_statistics.update({
-            'sparsity_rate_for_sparsified_modules': sparsity_rate_for_sparsified_modules,
-            'sparsity_rate_for_model': sparsity_rate_for_model,
-            'mean_sparse_prob': mean_sparse_prob,
-            'target_sparsity_rate': self.loss.target_sparsity_rate,
-        })
-
         sparsity_levels = tf.keras.backend.batch_get_value(sparsity_levels)
         weights_percentages = [weights_number / total_weights_number * 100
                                for weights_number in weights_numbers]
         weights_percentages = tf.keras.backend.batch_get_value(weights_percentages)
         mask_sparsity = list(zip(mask_names, weights_shapes, sparsity_levels, weights_percentages))
-        raw_sparsity_statistics['sparsity_statistic_by_layer'] = []
+
+        # TODO(andrey-churkin): Why we use `mask_name` instead of `layer_name`?
```

Comment:
It's name from magnitude sparsity `raw_statistic` method. I used `wrapped_layer.name + '_rb_mask'`  name because it contains name of the layer but mask.name consist of `weight_attribute + '_mask'` thus less user friendly. mask_name not actually contains `mask.name`s but contains their user frendly names

Reply:
Seems like a bug because `NNCFWrapper` can contain multiple operations with weights.